install

```diff
@@ -9,6 +9,13 @@ update_config=2
 binary_arch=
 allow_legacy=
 shells="bash zsh fish"
+config_dir=~/.config/fzf
```

Comment:
Don't want to suddenly change the default directory after 4 and a half years. Please change it back to `~`.
(Also note that Travis CI build is failing because of the change.)

Reply:
I reverted the default directory to `~/`, but also added a check to see if the user manually created a `~/.config/fzf` directory, which I hope is fine since some users want their dotfiles in `~/.confg` but don't want to set extra environment variables.

Reply:
Sounds good to me.